doc/source/user_guide/reshaping.rst

```diff
@@ -728,14 +728,9 @@ Suppose we wanted to pivot ``df`` such that the ``col`` values are columns,
 ``row`` values are the index, and the mean of ``val0`` are the values? In
 particular, the resulting DataFrame should look like:
 
-.. note::
+.. ipython:: python
 
-   col   col0   col1   col2   col3  col4
-   row
-   row0  0.77  0.605    NaN  0.860  0.65
-   row2  0.13    NaN  0.395  0.500  0.25
-   row3   NaN  0.310    NaN  0.545   NaN
-   row4   NaN  0.100  0.395  0.760  0.24
```

Comment:
@oktaysabak you need to keep the actual table here (the goal is to show an desired result from an operation done on `df`, not to show `df` itself)

So basically only the `.. note::` should be changed into `.. code-block::`